Harvest.Web/Controllers/Api/RequestController.cs

```diff
@@ -79,7 +81,16 @@ public async Task<ActionResult> Approve(int projectId, [FromBody] RequestApprova
             if (await _dbContext.Projects.AnyAsync(p => p.Id == projectId && p.OriginalProject != null))
             {
                 // this was a change request that has been approved, so copy everything over to original and inActiveate change request
-                var changeRequestProject = await _dbContext.Projects.SingleAsync(p => p.Id == projectId);
+                var changeRequestProject = await _dbContext.Projects.SingleAsync(p => p.Id == projectId);
+
+                if (changeRequestProject.PrincipalInvestigator.Iam != currentUser.Iam)
+                {
+                    var staleDays = (int)((DateTime.UtcNow - changeRequestProject.LastStatusUpdatedOn).TotalDays);
+                    if (staleDays <= MinimumStaleDays)
+                    {
+                        return BadRequest("You are not the principal investigator for this project and it isn't stale enough.");

```

Comment:
Will the term "stale" be meaningful for anyone that might see it?

Reply:
Probably not, but it doesn't get displayed anywhere.